fsharp-backend/src/LibExecution/StdLib/LibFloat.fs

```diff
@@ -272,37 +272,33 @@ let fns : List<BuiltInFn> =
       sqlSpec = NotYetImplementedTODO
       previewable = Pure
       deprecated = NotDeprecated }
-// ; { name = fn "Float" "clamp" 0
-
-//   ; parameters = [Param.make "value" TFloat ""; Param.make "limitA" TFloat ""; Param.make "limitB" TFloat ""]
-//   ; returnType = TFloat
-//   ; description =
-//       "If `value` is within the range given by `limitA` and `limitB`, returns `value`.
-//        If `value` is outside the range, returns `limitA` or `limitB`, whichever is closer to `value`.
-//        `limitA` and `limitB` can be provided in any order."
-//   ; fn =
-
-//         (function
-//         | _, [DFloat v; DFloat a; DFloat b] ->
-//             let min, max = if a < b then (a, b) else (b, a) in
-//             ( match Float.clamp v ~min ~max with
-//             | Ok clamped ->
-//                 DFloat clamped
-//             | Error e ->
-//                 (* Since min and max are pre-sorted, this can only happen if min or max are NaN.
-//                  * TODO: eliminate NaNs so that this can't happen
-//                  * (at time of writing (f86edaa1c58c94e27186060ae4fe8745112dd0e5), NaNs can't be parsed,
-//                  * so this can't happen in practice) *)
-//                 let info =
-//                   [("a", Float.to_string a); ("b", Float.to_string b)]
-//                 in
-//                 Exception.code
-//                   ~info
-//                   ("Internal Float.clamp exception: " ^ Error.to_string_hum e)
-//             )
-//         | _ ->
-//             incorrectArgs ())
-//   ; sqlSpec = NotYetImplementedTODO
-//   ; previewable = Pure
-//   ; deprecated = NotDeprecated }
+    { name = fn "Float" "clamp" 0
+      parameters = [Param.make "value" TFloat ""; Param.make "limitA" TFloat ""; Param.make "limitB" TFloat ""]
+      returnType = TFloat
+      description =
+         "If `value` is within the range given by `limitA` and `limitB`, returns `value`.
+          If `value` is outside the range, returns `limitA` or `limitB`, whichever is closer to `value`.
+         `limitA` and `limitB` can be provided in any order."
+      fn =
+        (function
+        | _, [DFloat v; DFloat a; DFloat b] ->
+            let output = false in
+            let min, max = if a < b then (a, b) else (b, a) in
+            ( match Single.TryParse(Math.Clamp(v, min, max), output) with
```

Comment:
What is this for? Math.Clamp seems to return a double, and parsing it doesn't make sense. I'm also unclear on why we'd use Single instead of Double.